app/cdap/text/text-en.yaml

```diff
@@ -3189,9 +3189,9 @@ features:
       searchLabel: Search by batch pipeline name
       tab: Local pipelines
     table:
-      connected: Connected to Git
+      connected: Connected
```

Comment:
The chip in detail page is using this text, maybe you can change it to the below one

Reply:
got it, updated the other place